signer/tests/integration/transaction_coordinator.rs

```diff
@@ -3763,6 +3780,47 @@ async fn sign_bitcoin_transaction_withdrawals() {
         txid: StacksTxId::from([123; 32]),
         sender_address: PrincipalData::from(StandardPrincipalData::transient()).into(),
     };
+
+    // Now we should manually put withdrawal request to Emily, pretending that
+    // users did it.
+    assert!(
+        testing_emily_client::apis::withdrawal_api::create_withdrawal(
+            &test_emily_config,
+            testing_emily_client::models::CreateWithdrawalRequestBody {
+                amount: withdrawal_request.amount,
+                parameters: Box::new(testing_emily_client::models::WithdrawalParameters {
+                    max_fee: withdrawal_request.max_fee,
+                }),
+                recipient: withdrawal_request.recipient.to_string(),
+                request_id: withdrawal_request.request_id,
+                sender: withdrawal_request.sender_address.to_string(),
+                stacks_block_hash: withdrawal_request.block_hash.to_string(),
+                stacks_block_height: db
+                    .get_stacks_block(&stacks_chain_tip)
+                    .await
+                    .unwrap()
+                    .unwrap()
+                    .block_height,
+            }
```

Comment:
nit: we can probably pass a const for `stacks_block_height` if we don't need the real height

Reply:
Makes sense. Can we move the `db.get_stacks_block...block_height` outside, for readability?